### PR TITLE
Use Conda and WinRPM

### LINKED ISSUE #29: Use Conda.jl for binary dependencies
See, https://github.com/Luthaf/Conda.jl

### PR DESCRIPTION
**Pull request by isuruf**
Depends on #32, #33

Fixes #29

### CODE REVIEW

**Comment by isuruf** [comment]
@spencerlyon2, would you mind reviewing this PR?

File: deps/build.jl [hunk 1]
```diff
@@ -5,58 +5,59 @@ using Compat
...
-@osx_only begin
-    if Pkg.installed("Homebrew") === nothing
-        print("Homebrew package not installed, please run Pkg.add(\"Homebrew\") to use it to download dependencies")
-    else
-        using Homebrew
-        provides(Homebrew.HB, "gmp", [libgmp], os = :Darwin)
-    end
+prefix=joinpath(BinDeps.depsdir(symengine), "usr")
+
+xx(t...) = (OS_NAME == :Windows ? t[1] : (OS_NAME == :Linux || length(t) == 2) ? t[2] : t[3])
```
Comment (spencerlyon2) [inline deps/build.jl]: What is `xx`?

Comment (isuruf) [inline deps/build.jl]: It's a simple function to do different things for different OSs. It's used below. (Copied it from here, https://github.com/JuliaGraphics/Cairo.jl/blob/master/deps/build.jl#L102)

File: deps/build.jl [hunk 2]
```diff
@@ -5,58 +5,59 @@ using Compat
 group = library_group("symengine")
 
 deps = [
```
Comment (spencerlyon2) [inline deps/build.jl]: This doesn't seem to be used anywhere, it is necessary?

**Comment by spencerlyon2** [comment]
The conda build seems to work for OS X if we change ccall syntax from

```julia
ccall((:fun_name, :libsymengine), ...)
```

to 

```julia
ccall((:fun_name, symengine), ...)
```

Where `symengine` is defined in deps/deps.jl

**Comment by isuruf** [comment]
Closed in favour of #35

File: deps/build.jl (hunk 1 above)
Comment (certik) [inline deps/build.jl]: @isuruf you could add your comment as an actual comment in the file.

**Pull request closed without merging**